pkgs/top-level/all-packages.nix

```diff
@@ -9108,6 +9108,10 @@ with pkgs;
   hash-slinger = callPackage ../tools/security/hash-slinger { };
 
   haskell-language-server = callPackage ../development/tools/haskell/haskell-language-server/withWrapper.nix { };
+  haskell-language-server-ghc-9_0 = callPackage ../development/tools/haskell/haskell-language-server/withWrapper.nix { supportedGhcVersions = [ "90" ]; };
+  haskell-language-server-ghc-9_2 = callPackage ../development/tools/haskell/haskell-language-server/withWrapper.nix { supportedGhcVersions = [ "92" ]; };
+  haskell-language-server-ghc-9_4 = callPackage ../development/tools/haskell/haskell-language-server/withWrapper.nix { supportedGhcVersions = [ "94" ]; };
+  haskell-language-server-ghc-9_6 = callPackage ../development/tools/haskell/haskell-language-server/withWrapper.nix { supportedGhcVersions = [ "96" ]; };
```

Comment:
Not that I have a strong opinion either way, but in the Haskell package sets use a format like `ghc94` instead of `ghc-9_4`.  It might make more sense to keep the same format of `ghc94`?

On the flip side, it feels like most packages in `all-packages.nix` use a format similar to `ghc-9_4` for version-specific attributes, so maybe a format like `haskell-language-server-ghc-9_4` is better?